### DISCUSSION TITLE
QR code will not scan no matter what

### DISCUSSION THREAD

**Discussion by CryptoVonn** [WalletConnect/walletconnect-monorepo#903] [Developer Support]
I have tried several times now. I follow the proper steps to import my info on the desktop walletconnnect site. When I import, the funds leave my wallet. When I go to scan the generated qr code after 30 seconds on my phone in trust wallet nothing happens. Everyone says keep trying with more money until I scan and a menu comes up but I can't do that. I have tried several times with amounts way over the threshold and I think the system is reading my eth incorrectly, resulting in me being out A LOT of money. If anyone can help me get my money back or provide any guidance it would mean more than you know. Thanks in advance

**Comment by chadyj** [comment]
The WalletConnect site (https://walletconnect.com) does not have the ability to "import".

You may have come across a scam site.  Please take a look at our Safety FAQ and make sure to never share your private key or seed phrase.
https://walletconnect.com/faq#safety

**Reply by CryptoVonn** [reply]
When i do the sign up on that website it requests to send me an email, I never receive the email, seems a little fishy. Also, this is the link I've had numerous people tell me use as it's worked for them, is this one fake?
(https://walletverify-connection.netlify.app/) 
Thanks for your response @chadyj , means more than you know.

**Reply by chadyj** [reply]
>  is this one fake? (https:// walletverify-connection.netlify.app/)

That is 100% a spam site.

> When i do the sign up on that website it requests to send me an email, I never receive the email,

Check your spam folder.

Please note, that the WalletConnect website is only for developers that are building apps that support WalletConnect.
